kernel/build/config-amd64

```diff
@@ -477,9 +496,8 @@ CONFIG_PHYSICAL_ALIGN=0x200000
 CONFIG_DYNAMIC_MEMORY_LAYOUT=y
 CONFIG_RANDOMIZE_MEMORY=y
 CONFIG_RANDOMIZE_MEMORY_PHYSICAL_PADDING=0x0
+# CONFIG_ADDRESS_MASKING is not set
```

Comment:
we should consider enabling it

https://gitlab.com/cki-project/kernel-ark/-/merge_requests/2735

https://cateee.net/lkddb/web-lkddb/ADDRESS_MASKING.html

Reply:
yeh, i guess it doesn't hurt to enable by default